qgreenland/util/luigi/tasks/pipeline.py

```diff
@@ -163,14 +163,14 @@ def requires(self):
             src_filepath=PROJECT_DIR / 'doc' / '_pdf' / 'QuickStartGuide.pdf',
             dest_relative_filepath='QuickStartGuide.pdf',
         )
+        yield AncillaryFile(
```

Comment:
I don't think we should merge this change in to main. We discussed leaving it on its own branch so that the hack would only exist in one release and would not need to be reverted. Is that still agreeable?

Reply:
yes that makes sense to me, I didn't realize this change was on here